css/style.css

```
@@ -254,4 +247,157 @@ footer {
 .footer-social img {
   width: 20px;
   height: 20px;
+}
+
+@media (max-width: 1199px) {
+  header nav {
+    padding: 9px 16px;
+  }
+  
+  .banner {
+    padding: 0;
+    display: block;
+    height: 90vh;
+    background-size: 120%;
+    text-align: center;
+  }
+
+  .banner-wrap {
+    display: block;
+  }
+
+  .banner-title {
+    margin: 0;
+  }
+
+  .banner-title h2 {
+    padding-top: 84px;
+  }
+
+  #topBanner h2 br {
+    display: none;
+  }
+
+  .main-content {
+    display: block;
+    padding: 24px 24px 16px 24px;
+  }
+
+  .content-wrap {
+    display: block;
+    margin-bottom: 52px;
+  }
+
+  .content-item {
+    display: block;
+    background-color: var(--white);
+  }
+
+  .content-wrap img {
```

❗️ 수정요청
반응형 작업을 하시면서 이미지에 width, height값이 모두 존재하니 이미지가 기존 비율을 유지하지 못하고 왜곡되어서 보이게 됩니다.
width 속성만 남겨두시고 height 속성을 제거해주시면 기존 비율을 유지하면서 이미지가 렌더링됩니다~

```suggestion
  .content-wrap img {
    height: auto; // auto로 하셔도 비율을 유지하며 height를 차지합니다.
```